### PR TITLE
target_sparsity_level and target_pruning_level statistics were added

### PR DESCRIPTION
**Pull request by andrey-churkin**

### CODE REVIEW

**Review by daniil-lyakhov — APPROVED** [review]

File: nncf/common/sparsity/statistics.py [hunk 1]
```diff
@@ -148,19 +159,20 @@ class RBSparsityStatistics(Statistics):
...
+                 target_sparsity_level: float,
                  mean_sparse_prob: float):
         """
         Initializes statistics of the RB-sparsity algorithm.
 
         :param model_statistics: Statistics of the sparsified model.
         :param masks_consistency: TODO
-        :param target_level: TODO
+        :param target_sparsity_level: Sparsity level which was applied
+            by the algorithm.
```
Comment (vshampor) [inline nncf/common/sparsity/statistics.py:170]: Wording must be changed. A target level is a level which, in general, has not yet been completely applied by the algo; the "sparsity level which was applied by the algorithm" is a *current* level. 

Suggest: "A target level of sparsity for the algorithm to ultimately achieve". Same in other places.

**Review by vshampor — CHANGES_REQUESTED** [review]

File: nncf/common/sparsity/statistics.py (hunk 1 above)
Comment (andrey-churkin) [reply nncf/common/sparsity/statistics.py:170]: I suggest a different formulation:
"A target level of the sparsity for the algorithm for the current epoch."

Comment (andrey-churkin) [reply nncf/common/sparsity/statistics.py:170]: @alexsu52 @vshampor What do you think?

Comment (vshampor) [reply nncf/common/sparsity/statistics.py:170]: Ok, it can be worded so.

Comment (andrey-churkin) [reply nncf/common/sparsity/statistics.py]: I fixed it.

**Review by vshampor — APPROVED** [review]

**Pull request merged**